docs/throttle/README.md

```diff
@@ -5,9 +5,17 @@ MinIO server allows to throttle incoming requests:
 - limit the number of active requests allowed across the cluster
 - limit the wait duration for each request in the queue
 
-These values are enabled using environment variables *only*.
+These values are enabled using server's configuration or environment variables.
 
-## Configuring connection limit
+## Parameters
+
+`api.request_max` or `MINIO_API_REQUESTS_MAX`: The maximum number of S3 requests that the server/cluster will process concurrently.
```

Comment:
There is no such thing as api.request_max @vadmeste either you need to use `mc admin config set` command fully and appropriately